src/GitHub.VisualStudio/GitHub.VisualStudio.vsct

```diff
@@ -37,6 +37,10 @@
       <Group guid="guidGitHubToolbarCmdSet" id="idGitHubToolbarMenuGroup1" priority="0x0501">
         <Parent guid="guidGitHubToolbarCmdSet" id="idGitHubToolbar" />
       </Group>
+
+      <Group guid="guidCreateGistCommandPackageCmdSet" id="GistContextMenuGroup" priority="0x0600">
+        <Parent guid="guidSHLMainMenu" id="IDM_VS_CTXT_CODEWIN"/>
```

Comment:
CODEWIN is probably why the menu entry doesn't show up in non code windows. Some code windows are managed differently and have different GUIDs, so this is annoying.

I've pushed a couple of things to master to deal with this. There's a context menu group there now that hooks up into different types of code windows, so you can parent a command to it and it will show up everywhere.